sdk-base/src/main/java/com/mapbox/maps/MapboxExceptions.kt

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.maps
+
+/**
+ * Mapbox exception class flagging map related [RuntimeException] did occur.
+ */
+class MapboxMapException(
+  exceptionText: String?
+) : RuntimeException(exceptionText)
+
+/**
+ * Mapbox exception class flagging view annotation related [RuntimeException] did occur.
+ */
+class MapboxViewAnnotationException(
```

Comment:
shall we move the plugin-specific exception to the plugin package?

Reply:
I did it initially but imho better have them all in one centralized place. Moreover view annotation is not a plugin 😉 
Guess package `com.mapbox.maps` is the correct place .